src/spectro_inlets_quantification/sensitivity.py

```diff
@@ -1137,3 +1140,59 @@ def plot_F_vs_f(
             F_fit = self.alpha * f_fit
             ax.plot(f_fit, F_fit, "k--")  # type: ignore
         return ax
+
+
+def append_sensitivity_factors(*args):
+    """Recursively append elements in a list"""
+    sf = append_two_sensitivity_factors(args[0], args[1])
+    if len(args) == 2:
+        return sf
+    remaining_args = [sf] + list(args[2:])
+    return append_sensitivity_factors(*remaining_args)
+
+
+def append_two_sensitivity_factors(
+        sf_1: Union[SensitivityFactor, SensitivityList],
+        sf_2: Union[SensitivityFactor, SensitivityList]
+) -> SensitivityList:
+    """This should be incorporated into __add__ of SensitivityFactor and SensitivityList
+
+    Appends all the sensitivity factors in sf_1 and sf_2, which can each be a single
+    SensitivityFactor (or CalPoint) or a SensitivityList (or Calibration).
+
+    Args:
+        sf_1 (SensitivityFactor or SensitivityList): The first sensitivity factor or
+            list thereof.
+        sf_2 (SensitivityFactor or SensitivityList): The second sensitivity factor or
+            list thereof
+
+    Returns SensitivityList: a `SensitivityList` containing all of the sensitivity
+        factors in `sf_1` and `sf_2`. This will be a `Calibration` unless both `sf_1`
+        and `sf_2` are both themselves plain `SensitivityList`s
+    """
+    from .calibration import Calibration
+
+    if isinstance(sf_1, SensitivityList):
+        if isinstance(sf_2, SensitivityList):
+            return sf_1 + sf_2  # this is implemented in spectro_inlets_quantification
+        elif isinstance(sf_2, SensitivityFactor):
+            sf_list = sf_1.sf_list + [sf_2]
+            obj_as_dict: Any = sf_1.as_dict()
```

Comment:
There actually is a type for this, see `SensitivityFactor.as_dict`

Reply:
Yeah, but that type wasn't working for me... I couldn't figure out why not.